src/main/assembly/dist/bag/metadata/agreements/2018/12/agreements.xsd

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (C) 2012 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:dcterms="http://purl.org/dc/terms/" elementFormDefault="qualified"
+    xmlns:am="http://easy.dans.knaw.nl/schemas/bag/metadata/agreements/"
+    targetNamespace="http://easy.dans.knaw.nl/schemas/bag/metadata/agreements/">
+    <xs:import namespace="http://purl.org/dc/terms/" schemaLocation="http://dublincore.org/schemas/xmls/qdc/2008/02/11/dcterms.xsd"/>
+    <xs:annotation>
+        <xs:documentation>
+            This schema specifies a metadata-format for describing the agreements between the depositor and DANS-KNAW.
+            It contains information about the existence of personal data within the deposit as well.
+
+            Created 2018-05-01
```

Comment:
could you add a change log, just like in https://github.com/DANS-KNAW/easy-schema/blob/5f6d4ff4445ad7887bb22d7fc7bcb99c91850769/src/main/assembly/dist/bag/metadata/files/files.xsd#L31-L35

Reply:
what about https://github.com/DANS-KNAW/easy-specs/blob/master/dans-bagit-profile/v0/dans-bagit-profile-v0.md#33-metadataagreementsxml
it refers to version 2018/05